sdk/cognitivelanguage/ai-language-text/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
I think when creating a new package we try to have the new version be greater than the old version. Not sure if this was written into guidelines, but might be worth checking.

Reply:
Good point, I started an email thread about this and can address the issue in another PR.